src/csharp/build_packages.bat

```diff
@@ -30,7 +30,7 @@
 @rem Builds gRPC NuGet packages
 
 @rem Current package versions
-set VERSION=1.0.0-pre1
+set VERSION=1.0.0-pre2
 set PROTOBUF_VERSION=3.0.0-beta3
```

Comment:
But yes, we should wait before releasing pre2 that everything has been published. Doesn't affect the version bump however :)